docs/src/userguide/language_basics.rst

```diff
@@ -274,25 +453,50 @@ parameters and a new reference is returned).
 
 The name object can also be used to explicitly declare something as a Python
 object. This can be useful if the name being declared would otherwise be taken
-as the name of a type, for example,::
+as the name of a type, for example,
 
-    cdef ftang(object int):
-        ...
+.. tabs::
+
+    .. group-tab:: Pure Python
+
+        .. code-block:: python
+
+            @cfunc
+            def ftang(int: object):
+                ...
+
+    .. group-tab:: Cython
+
+        .. code-block:: cython
+
+            cdef ftang(object int):
+                ...
 
-declares a parameter called int which is a Python object. You can also use
+declares a parameter called ``int`` which is a Python object. You can also use
 object as the explicit return type of a function, e.g.::
 
     cdef object ftang(object int):
         ...
 
+.. note:: Currently, Cython contains a bug not allowing returning ``object`` in
+    pure python from a C function.
```

Comment:
Is there an issue for this that could be referenced here? That would make it easy to find out if the docs become outdated (by seeing that the referenced issue has been closed).

Reply:
The bug referenced is following: #2529. Not sure whether we want there references to bugs... (I understand that it adds visibility)